lib/versioning/VersionID.js

```diff
@@ -9,12 +9,12 @@
 // the lengths of the components in bytes
 const LENGTH_TS = 14; // timestamp: epoch in ms
 const LENGTH_SEQ = 6; // position in ms slot
-const LENGTH_SITE = 5; // site identifier
+const LENGTH_TOKEN = 5; // replication group identifier
```

Comment:
I increased to 7 bytes for now, to give just a little more room for naming the replication group.